assets/chezmoi.io/docs/reference/target-types.md

```diff
@@ -108,7 +108,8 @@ extensions. Other extensions require an interpreter, which must be in your
 The default script interpreters are:
 
 | Extension | Command      | Arguments |
-| --------- | ------------ | --------- |
+| --------- | ------------ |-----------|
+| `.nu`     | `nushell`    | *none*    |
```

Comment:
Is the command `nu` or `nushell`? In this line the command is described as `nushell` but in `util_windows.go` the default is set to `nu`.

Reply:
The command is `nu`, I was thrown by `powershell`, which changed to `pwsh` in later versions.